### PR TITLE
feat: add typings blog

### PR DESCRIPTION
**Pull request by nikku**
This is a late introduction to our type definitions (shipped with bpmn-js@13+).

Depends on https://github.com/bpmn-io/bpmn-js-examples/pull/268 to be merged.

### CODE REVIEW

File: src/blog/2024/001-bpmn-js-type-definitions.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+ * @type { ElementRegistry }
+ */
+const elementRegistry = viewer.get('elementRegistry');
+
+const element = elementRegistry.get('MY_TASK');
+
+console.log(element.id); // MY_TASK
+```
+
+Check out the [full types example](https://github.com/bpmn-io/bpmn-js-examples/blob/main/types) to learn more about bpmn-js type declarations and their use cases.
```
Comment (philippfromme) [inline src/blog/2024/001-bpmn-js-type-definitions.md:64]: I assume we're going to create this example?

Comment (nikku) [reply src/blog/2024/001-bpmn-js-type-definitions.md:64]: Cf. https://github.com/bpmn-io/bpmn-js-examples/pull/268

Comment (philippfromme) [reply src/blog/2024/001-bpmn-js-type-definitions.md:64]: 😅

**Review by philippfromme — APPROVED** [review]
Awesome! 🎖️

**Pull request merged**